wdl-analysis/src/lib.rs

```diff
@@ -14,6 +14,10 @@ mod graph;
 mod queue;
 mod rayon;
 mod scope;
+mod stdlib;
```

Comment:
I'm unsure of whether these should live in `wdl-analysis` permanently. But I'm okay with them living here for now.

Reply:
I 100% agree. I figure that this will be a good candidate to extract out into a common type system crate when we have need to do so (i.e. implementing an executor).